deploy/cdk_exec_policy_tooling/cdkExecPolicy_tooling.yaml

```diff
@@ -0,0 +1,590 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: Custom least privilege IAM policy for bootstrapping tooling and deployment accounts
+Parameters:
+  AwsAccountId:
+    Description: AWS AccountId of the tooling account.
+    Type: String
+  DeploymentAWSAccountId:
+    Description: AWS AccountId of the deployment account.
+    Type: String
+  PolicyNamePrefix:
+    Description: IAM policy name prefix. 
+    Type: String
+    Default: 'cdk-boot-exec-lpp'
+  EnvironmentResourcePrefix:
+    Description: The resource prefix value of the dataall environment. It MUST match the resource prefix that we use when we create the environment.
+    Type: String
+Resources:
+  CDKBootStrapPolicy1:
+    Type: 'AWS::IAM::ManagedPolicy'
+    Properties:
+      ManagedPolicyName: !Sub '${PolicyNamePrefix}-1'
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - 
+              Sid: VisualEditor10
+              Effect: Allow
+              Action:
+                  - 'cloudformation:DeleteStack'
+                  - 'cloudformation:TagResource'
+                  - 'cloudformation:CreateChangeSet'
+                  - 'cloudformation:UntagResource'
+              Resource:
+                - !Sub 'arn:aws:codeartifact:*:${AwsAccountId}:domain/${EnvironmentResourcePrefix}-*'
```

Comment:
Do we require all these resources for only the set of CloudFormation Actions listed in this statement?